app/src/main/java/com/example/keybindhelper/ui/keybind/KeybindFragment.kt

```diff
@@ -15,81 +15,98 @@ import com.example.keybindhelper.R
 import com.example.keybindhelper.RecyclerViewAdapters.GroupAdapter
 import com.example.keybindhelper.dao.CurrentProjectManager
 
-
 class KeybindFragment : Fragment() {
-
-
-
     override fun onCreateView(
         inflater: LayoutInflater,
         container: ViewGroup?,
         savedInstanceState: Bundle?,
     ): View {
-
-        val root: View = LayoutInflater.from(this.context).inflate(R.layout.fragment_keybind,container,false)//binding.root
-
-        val mainActivity=activity as MainActivity;
-        if(mainActivity.Menu==null)
-            mainActivity.onMenuInit=(object:MainActivity.MenuInitialized{
+        // Alex code review: https://kotlinlang.org/docs/coding-conventions.html#wrap-chained-calls
```

Comment:
Big no-no: do not put conversation in source code comments.  That's clutter.  Put the conversation here, in the Files Changed section of the pull request, where you can have a dialog with other developers.

It looks like you are indicating changes for your review.  You can see that automatically in GitHub, as your review is a separate commit.